charts/gitops-runtime/templates/_components/cap-app-proxy/_deployment.yaml

```diff
@@ -57,6 +57,8 @@ spec:
         ports:
         - name: http
           containerPort: 8080
+        - name: http-metrics
+          containerPort: 9100
```

Comment:
not directly related. needed if we want to turn on the otel env vars for app-proxy, and add a serviceMonitor for it